packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -772,7 +772,7 @@ oie.registration.form.title = 》Šîĝñ ûþ ヸ€홝한Ӝฐโ《
 oie.registration.form.submit = 》Šîĝñ Ûþ ヸ€홝한Ӝฐโ《
 oie.user.profile.lastname = 》Ļåšţ ñåɱé ئ䀕ヸ€홝한Ӝฐโ《
 oie.user.profile.firstname = 》Ƒîŕšţ ñåɱé ⾼ئ䀕ヸ€홝한Ӝฐโ《
-oie.user.profile.primary.email = 》Þŕîɱåŕý éɱåîļ ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.user.profile.primary.email = 》Éɱåîļ 홝한Ӝฐโ《
```

Comment:
This diff should go away once my other PR is merged - https://github.com/okta/okta-signin-widget/pull/1987